hack/util

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+: ${PREFER_DOCKER=}
+: ${PREFER_LEGACY=}
+
+newerEqualThan() { # $1=minimum wanted version $2=actual-version
+  [ "$1" = "$(echo -e "$1\n$2" | sort -V | head -n 1)" ]
+}
+
+buildmode="legacy"
+if [ "$PREFER_DOCKER" != "1" ] && [ "$PREFER_LEGACY" != "1" ] && buildctl debug workers 2>/dev/null >/dev/null; then
```

Comment:
I've just noticed that this `buildctl debug workers` hangs indefinitely (or at least a very long time) if you have `buildctl` in your path but are not currently running `buildkitd` (not uncommon in a buildkit dev environment I guess?).

Once I control-C it then it says:
```
error: failed to dial "unix:///run/buildkit/buildkitd.sock" . make sure buildkitd is running: context canceled
```
The code seems to be using `client.WithFailFast()` in the right place so I'm not sure what is going on.

Reply:
Did you update `buildctl`? I think I fixed this last week.